src/http-spec.ts

```diff
@@ -181,7 +181,7 @@ export interface IBearerSecurityScheme extends ISecurityScheme {
 
 export interface IBasicSecurityScheme extends ISecurityScheme {
   type: 'http';
-  scheme: 'basic';
+  scheme: 'basic' | 'digest';
```

Comment:
Should we make digest it's own interface `IDigestSecurityScheme` like we're doing for the other `http` securities? https://github.com/stoplightio/types/blob/master/src/http-spec.ts#L176-L180

Reply:
Not too sure it's worth it, since there's no difference in the fields per se. The other ones have different fields and so a tagged union makes sense.

Reply:
Sure 👍